Server/Services/ILocalTempFileLocks.cs

```diff
@@ -14,15 +16,44 @@ public interface ILocalTempFileLocks
     ///   to avoid multiple places using the same temporary folder at once
```

Comment:
This summary should say to call `LockAsync` with the result path, as "lock the returned string" might lead to misunderstanding otherwise.

Reply:
Done, not yet committed.